operators/advanced_network/Dockerfile

```diff
@@ -43,10 +99,11 @@ RUN if [ "${TARGETARCH}" = "amd64" ]; then \
     && DOCA_REPO_LINK=https://linux.mellanox.com/public/repo/doca/${DOCA_VERSION}/${DISTRO}/${DOCA_ARCH} \
     && echo "Using DOCA_REPO_LINK=${DOCA_REPO_LINK}" \
     && LOCAL_GPG_KEY_PATH="/usr/share/keyrings/mellanox-archive-keyring.gpg" \
-    && curl -fsSL ${DOCA_REPO_LINK}/GPG-KEY-Mellanox.pub | gpg --dearmor | tee ${LOCAL_GPG_KEY_PATH} \
+    && curl -fsSL ${DOCA_REPO_LINK}/GPG-KEY-Mellanox.pub | gpg --dearmor | tee ${LOCAL_GPG_KEY_PATH} > /dev/null \
     && echo "deb [signed-by=${LOCAL_GPG_KEY_PATH}] ${DOCA_REPO_LINK} ./" | tee /etc/apt/sources.list.d/mellanox.list
 
 # APT installs
+# - cublas, cufft, cusolver, curand dev libs: for matx
```

Comment:
Is MatX already an ANO dependency or is this in anticipation of upcoming HSDK features?

Reply:
already is